### ISSUE TITLE
ghost shadow left on the window after enter and exit full screen when displaying very long lines

### ISSUE THREAD

**Issue by volksvagen** [microsoft/terminal#15537]
### Windows Terminal version

1.17.11461.0

### Windows build number

10.0.23475.1000

### Steps to reproduce

1. Win + Run to open wt.exe
2. echo a very long string to exceeds max column of current window, until typing to line break
3. press Alt + Enter to go full screen
4. press Alt + Enter again to exit full screen
5. now we can see some ghost shadow left on the screen, next to restored wt.exe window, usually to the right
6. this shadow is always on top, top of desktop, and top of other background applications
7. this bug does **NOT** happen on Windows 10, but **ONLY** on Windows 11
8. this bug happens in both user and administrative mode
9. click anywhere outside of wt.exe, the ghost shadow disappears
![image](https://github.com/microsoft/terminal/assets/30553808/2321f05e-dbde-4e48-bd38-d21ac41fd450)
![image](https://github.com/microsoft/terminal/assets/30553808/96ee94b4-7e91-436a-8575-bc1665fe1270)
![image](https://github.com/microsoft/terminal/assets/30553808/fa2d50e4-787a-441a-8b4f-8b1aea1e83b8)
![dc9a58ba2bb8e003715a1179de1efbf](https://github.com/microsoft/terminal/assets/30553808/09a141b8-af2f-4ce2-9b68-422eb438e5e2)

### Expected Behavior

no ghost shadow should be left on the screen after exiting full screen mode.

### Actual Behavior

some ghost shadow is left on the screen after switching full screen mode.

**Comment by lhecker** [comment]
Thanks for the report! This is effectively a duplicate of #14865. While that issue says "Intel Iris Xe" and looks a bit different, the end result is the same: There's "ghosting" outside of the window. I used to have the same issue with my GPU (RTX 3080). What GPU do you have?

It's almost certainly a bug in DWM and we aren't entirely sure what triggers it and/or how to fix it. It's probably not due to Windows Terminal. Since updating to Windows 11 22631 (Windows Insider Beta channel) I personally didn't have this issue anymore.

I'll close this issue, because it's the same issue as #14865. /dup #14865
Please feel free to reopen it, if you disagree!

**Comment by volksvagen** [comment]
Hi @lhecker, thank you for the confirmation, my GPU is "NVIDIA Quadro T2000", and my Windows 11 version is "Windows 11 Insider Preview 22H2 23475.1000", "Windows Feature Experience Pack 1000.23475.1000.0", I have no idea whose Windows is newer here.

Any tips to upgrade my version to yours? Windows 11 22631 (Windows Insider Beta channel)

I checked my windows version, it says already latest.

I also tried Windows 11 upgrade assistant, also said latest.
